packages/story-editor/src/app/story/useStoryReducer/reducers/utils.ts

```diff
@@ -14,15 +14,22 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import type { Element, Page } from '@googleforcreators/elements';
+import type { StoryAnimation } from '@googleforcreators/animation';
+
 /**
  * Internal dependencies
  */
 import { LAYER_DIRECTIONS } from '../../../../constants';
 import { ELEMENT_RESERVED_PROPERTIES } from '../types';
 import objectWithout from '../../../../utils/objectWithout';
+import type { ElementUpdater } from '../../../../types';
 export { objectWithout };
 
-export function intersect(first, ...rest) {
+export function intersect(first: string[], ...rest: string[][]) {
```

Comment:
This should probably be generic - just update the typing and the rest can remain:

```suggestion
export function intersect<T>(first: T[], ...rest: T[][]) {
```